site/docs/v1/tech/identity-providers/google.adoc

```diff
@@ -141,5 +142,21 @@ include::docs/v1/tech/identity-providers/_login-api-integration.adoc[]
 :identity_provider_fragment!:
 :identity_provider_path!:
 
+== Custom Parameters
+
+Google sometimes requires custom URL parameters when starting a login to access certain functionality. Examples include: 
+
+* `access_type=offline` if you want a refresh token from Google.
+* `prompt=select_account` if you want to force an account selection screen.
+
+There are more https://developers.google.com/identity/protocols/oauth2/openid-connect#advancedtopics[options covered here].
+
+When you need such custom parameters, the Google Identity Provider won't work. Instead, do the following:
+
+* Create an link:/docs/v1/tech/identity-providers/openid-connect/[OpenId Connect Identity Provider].
+* Set [field]#Discover endpoints# to be `false`.
+* Enter the endpoints manually from the https://developers.google.com/identity/protocols/oauth2/openid-connect#discovery[Discovery document].
+* Append whatever additional parameters are needed to the [field]#Authorization endpoint# value. For example, you might end up with something like `\https://accounts.google.com/o/oauth2/v2/auth?access_type=offline&prompt=select_account` .
```

Comment:
As this is a valid workaround -- should we open a feature request to have the existing Google typed idp extended?

Reply:
Sure. I'll do that.